#2125 Semantic Browser improve data queries

georgiana-ecobici opened the pull request:
- Updates on SB to see missing details about some elements

Resolves #2125 


Change-Id: Iad46cc836a0c2a439d19ae693342c844fd9df09c
Signed-off-by: Georgiana Ecobici <georgiana-elena.ecobici@thalesgroup.com>

pdulth reviewed (changes requested):
Hi, 
Can you separate both issues into two dedicated commits issues on github as they are unrelated issues ?
-     Delete Dialog see referencing elements on opening the dialog
-     Updates on SB to see missing details about some elements
Otherwise, looks ok, missing junits

georgiana-ecobici commented:
> 
> 
> Hi,
> Can you separate both issues into two dedicated commits issues on github as they are unrelated issues ?
> 
>     * ```
>       Delete Dialog see referencing elements on opening the dialog
>       ```
> 
>     * ```
>       Updates on SB to see missing details about some elements
>       ```
> 
> 
> Otherwise, looks ok, missing junits

Moved the Delete Dialog impact analysis bug fix to a separate commit.
Added Junits for this commit.
